Paket.LoadingScripts/LoadingScriptsGenerator.fs

```diff
@@ -0,0 +1,156 @@
+        (fun (p:PackageResolver.ResolvedPackage) -> p.Name) 
+        (fun p -> p.Dependencies |> Seq.map (fun (n,_,_) -> n)) 
+    let getPackageOrderFromDependenciesFile (depFile:string) =
+        let depFile = LockFileParser.Parse (System.IO.File.ReadAllLines depFile)
+        depFile
+        |> Seq.map (fun p -> p.GroupName, getPackageOrderResolvedPackage p.Packages)
+        |> Map.ofSeq
+        // generateScriptsFromDependenciesFile generateIncludeScript depFile
+
+    let testOrdering =
+      let testData =
+          [ { PackageResolver.ResolvedPackage.Name = PackageName("Test1")
+              PackageResolver.ResolvedPackage.Version = SemVer.Parse "1.0.0"
+              PackageResolver.ResolvedPackage.Dependencies =
+                Set.empty
+                |> Set.add(
+                    PackageName("other"), 
+                    VersionRequirement(VersionRange.Specific (SemVer.Parse "1.0.0"), PreReleaseStatus.No),
+                    Paket.Requirements.FrameworkRestrictions.AutoDetectFramework)
+              PackageResolver.ResolvedPackage.Unlisted = false
+              PackageResolver.ResolvedPackage.Settings = Requirements.InstallSettings.Default
+              PackageResolver.ResolvedPackage.Source = PackageSources.PackageSource.NuGetV2 { Url = ""; Authentication = None } }
+            { Name = PackageName("other")
+              Version = SemVer.Parse "1.0.0"
+              Dependencies = Set.empty
+              Unlisted = false
+              Settings = Requirements.InstallSettings.Default
+              Source = PackageSources.PackageSource.NuGetV2 { Url = ""; Authentication = None } }
+          ]
+      let result =
+        getPackageOrderResolvedPackage testData
+        |> List.map (fun p -> p.Name)
+
+      System.Diagnostics.Debug.Assert(
+        result = 
+          [  PackageName("other")
+             PackageName("Test1")
+          ] : bool)
+          
+      let result2 =
+        getPackageOrderResolvedPackage (testData |> List.rev)
+        |> List.map (fun p -> p.Name)
+
+      System.Diagnostics.Debug.Assert(
+        result2 = 
+          [  PackageName("other")
+             PackageName("Test1")
+          ] : bool)
+
+
+module ScriptGeneratingModule =
+  open System.IO
+  let getScriptName (package: PackageName) = sprintf "Include_%s.fsx" (package.GetCompareString())
+  let generateFSharpScript packagesOrGroupFolder (knownIncludeScripts:Map<PackageName, string>) (package: PackageResolver.ResolvedPackage) =
+    let packageFolder = Path.Combine (packagesOrGroupFolder, package.Name.GetCompareString())
+    let scriptFile = Path.Combine (packageFolder, getScriptName package.Name)
+    let relScriptFile = Path.Combine (package.Name.GetCompareString(), getScriptName package.Name)
+    let depLines =
+      package.Dependencies
+      |> Seq.map (fun (depName,_,_) -> sprintf "#load \"../%s\"" ((knownIncludeScripts |> Map.find depName).Replace("\\", "/")))
+
+    let dllFiles =
```

Comment:
I'm working on a more robust way to figure out the dll files to get those in correct order and leverage Paket.InstallModel.